lib/generators/react_on_rails/templates/redux/base/client/app/lib/middlewares/loggerMiddleware.js

```diff
@@ -12,7 +12,7 @@ export default function logger({ getState }) {
 
     // We can't console.log immutable objects out-of-the-box.
     const immutableState = getState();
-    const readableState = _.reduce(immutableState, (result, immutable, key) => {
+    const readableState = _.reduce(immutableState, (object, immutable, key) => {
       result[key] = immutable.toJS();
```

Comment:
Lodash docs: https://lodash.com/docs#reduce
The iteratee is bound to thisArg and invoked with four arguments: (accumulator, value, index|key, collection). 

Initial value is on line 17: `{}`

When you used `result` as your accumulator, it hid the parent one so it worked just fine. You could have called it pretty much anything in the outer closure.